rubixoscli/ffdevices.go

```diff
@@ -84,15 +84,3 @@ func (inst *Client) DeleteDevice(hostIDName, uuid string) (bool, error) {
 	}
 	return true, nil
 }
-
-func (inst *Client) FFGetPluginSchemaDevice(hostIDName, pluginName string) ([]byte, error) {
```

Comment:
what has this been replaced with? It was being used in https://github.com/NubeIO/commissioning-scripts-rubix-os/blob/cps/client-specific/cps/scripts/gateway_device_creation.go#L811

Reply:
It is supposed to copy all RubixCE `rubixoscli` into this project whenever we want to upgrade. And whenever we want to add something, we add it in RubixCE first and then copy it here. I've changed the RubixCE repo at first from this: https://github.com/NubeIO/rubix-ce/pull/1435 and resynced all contents in this project.

This is because:
- We take this RubixCE ROS CLIs as the single source of truth
- These CLIs will be used by other third parties as well so we can't give RubixCE access so we separated it out from there
- We could use `rubix-os-client` in RubixCE as well, but as it gets changed frequently we maintained it in two different places